app/src/main/res/values-cy/strings.xml

```diff
@@ -639,17 +665,11 @@
     <string name="biometric_auth_moved_too_quickly">Symudodd y bys yn rhy gyflym. Ceisiwch eto.</string>
 
     <!-- Title for the prompt to enable search suggestions -->
-    <string name="enable_search_suggestion_title">Hoffech chi agor awgrymiadau chwilio?</string>
+    <string name="enable_search_suggestion_title2">Dangos awgrymiadau chwilio?</string>
 
     <!-- Description for the prompt to enable search suggestions
-        %1$s will be replaced by the app name (e.g. Firefox Focus)
-        %2$s will be replaced with "Learn more" which will be a tappable link
-        using the string: "enable_search_suggestion_subtitle_learnmore" -->
-    <string name="enable_search_suggestion_subtitle">I dderbyn awgrymiadau chwilio, mae %1$s angen anfon yr hyn rydych yn ei deipio i\'r bar cyfeiriad i beiriant chwilio rydych wedi ei ddewis. %2$s.</string>
-
-    <!-- String to be placed inside the string: enable_search_suggestion_subtitle,
-        which will be a "Learn More" link. -->
-    <string name="enable_search_suggestion_subtitle_learnmore">Dysgu rhagor</string>
+        %1$s will be replaced by the app name (e.g. Firefox Focus) -->
+    <string name="enable_search_suggestion_description">I dderbyn awgrymiadau, mae %@ angen anfon yr hyn rydych chi’n ei deipio yn y bar cyfeiriad i’r peiriant chwilio.</string>
```

Comment:
@Delphine This string is wrong: https://pontoon.mozilla.org/cy/focus-for-android/mozilla-mobile/focus-android/app/src/main/res/values/strings.xml/?search=enable_search_suggestion_description&string=230895.

Again, not sure where the placeholder should be, maybe replacing "%@" ?

Reply:
@mcarare Fixed! I've reached out to the localizer before about this, not sure why there are still errors in placeholders. Have reached out again